paper-radio-group.html

```diff
@@ -120,6 +119,12 @@
       allowEmptySelection: {
         type: Boolean,
         value: false
+      },
+
+      disabled: {
+        type: Boolean,
+        reflectToAttribute: true,
```

Comment:
there's code in higher-up behaviors that check for `.hasAttribute('disabled')`, which is why I asked @hcarmona to add `reflectToAttribute`